### PR TITLE
adding new env variable

### PR DESCRIPTION
**Pull request by IsaacEldridge**

### CODE REVIEW

**Review by bjoson-mulesoft — DISMISSED** [review]

File: modules/ROOT/pages/install-manual.adoc [hunk 1]
```diff
@@ -207,6 +207,7 @@ The following variables can be defined based upon your organization's requiremen
 {sp} +
 {sp} +
 If the host operating system is RHEL 8, do not set to `false` or installation will fail. | `true`
+| `RTF_Private_Interface` | Allows private IP address to be discovered from the network interface. | `eth0`
```
Comment (digvijaysk19) [inline modules/ROOT/pages/install-manual.adoc]: This needs to be all caps. RTF_PRIVATE_INTERFACE

Comment (IsaacEldridge) [inline modules/ROOT/pages/install-manual.adoc]:
```suggestion
| `RTF_PRIVATE_INTERFACE` | Allows private IP address to be discovered from the network interface. | `eth0`
```

Comment (IsaacEldridge) [reply modules/ROOT/pages/install-manual.adoc]: Fixed, thanks

**Review by bjoson-mulesoft — APPROVED** [review]

**Review by digvijaysk19 — APPROVED** [review]
LGTM

**Pull request merged**